include/mptcpd/network_monitor.h

```diff
@@ -74,6 +74,60 @@ struct mptcpd_interface
         //@}
 };
 
+/**
+ * @struct mptcpd_nm_ops network_monitor.h <mptcpd/network_monitor.h>
+ *
+ * @brief Network monitor event tracking operations.
+ *
+ * A set of functions to be called when changes in network interfaces
+ * and addresses occur.
+ */
+struct mptcpd_nm_ops
```

Comment:
I'm not particularly fond of this `struct` name.  Perhaps something like `mptcpd_nm_events` might be better?